Hackney.Shared.Finance/Boundary/Response/ApiResponse.cs

```diff
@@ -0,0 +1,25 @@
+﻿
+namespace Hackney.Shared.Finance.Boundary.Response
+{
+    /// <summary>
+    /// API Response wrapper for all API responses
+    /// If a request has been successful this will be denoted by the statusCode
+    ///     Then the 'data' property will be populated
+    /// If a request has not been successful denoted
+    ///     Then the Error property will be populated
```

Comment:
The summary doesn't match the model.
Though, it's probably for the better. It may be best to stick to the standard responses dotnet itself gives for the errors. I don't think there's any value in having 1 bloated object that can contain both the error and the data.